WordPress/Docs/WP/DiscouragedConstantsStandard.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Discouraged Constants"
+    >
+    <standard>
+    <![CDATA[
+    The usage of WordPress Constants is discouraged.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: A function is used in the code to retrieve the theme directory.">
```

Comment:
This is not explicitly mentioned in #1722, but my understanding, based on checking the documentation from other sniffs, is that the description of the valid and invalid examples should be generic and describe what constitutes a valid example for this particular check that the sniff performs instead of describing the code sample provided.

Maybe here, the valid description should mention that the code does not use one of the discouraged WP constants or something like that?